eclass/systemd.eclass

```diff
@@ -242,19 +234,19 @@ systemd_install_serviced() {
 # @DESCRIPTION:
 # Deprecated in favor of tmpfiles.eclass.
```

Comment:
We should ban the systemd tmpfiles functions in EAPI 8.

Reply:
Should they be banned, or just removed?  I asked about [bug #740638](https://bugs.gentoo.org/740638) in the gentoo-devel e-mail: is there a removal date?

Reply:
Yeah, let's remove them in that case.

Reply:
Rebased with that removal in its own commit.